src/EditorFeatures/VisualBasicTest/Formatting/Indentation/SmartIndenterTests.vb

```diff
@@ -167,7 +167,7 @@ End Module
             AssertSmartIndent(
                 code,
                 indentationLine:=1,
-                expectedIndentation:=12)
+                expectedIndentation:=8)
```

Comment:
this result is better than before.  we at least align you to the previous comment, instead of indenting you an additional level.